redis/tests/support/mock_cluster.rs

```diff
@@ -135,10 +138,12 @@ impl cluster_async::Connect for MockConnection {
         _connection_timeout: Duration,
         _socket_addr: Option<SocketAddr>,
         _push_sender: Option<mpsc::UnboundedSender<PushInfo>>,
+        _disconnect_notifier: Option<Box<dyn DisconnectNotifier>>,
     ) -> RedisFuture<'a, (Self, Option<IpAddr>)>
     where
         T: IntoConnectionInfo + Send + 'a,
     {
+        let _ = _disconnect_notifier;
```

Comment:
? if you use it, then remove the `_` from the variable: `_disconnect_notifier`

Reply:
removed altogether